ras_party/controllers/party_controller.py

```diff
@@ -27,42 +27,46 @@
 
 @translate_exceptions
 @with_db_session
-def get_business_by_ref(ref, session):
+def get_business_by_ref(ref, session, verbose=False):
     """
     Get a Business by its unique business reference
     Returns a single Business
     :param ref: Reference of the Business to return
     :type ref: str
 
+    :param verbose: Verbosity of business details
+
     :rtype: Business
     """
     business = session.query(Business).filter(Business.business_ref == ref).first()
     if not business:
         raise RasError("Business with reference '{}' does not exist.".format(ref), status_code=404)
 
-    return business.to_business_dict()
+    return business.to_business_dict(summary=not verbose)
```

Comment:
would it be simpler to flip the logic for summary and just call it verbose too? Then we don't have to invert the logic half way through